src/main/resources/static/views/project/ProjectPage.html

```diff
@@ -1,4 +1,117 @@
-<div class="centered">
-    <h1>Here must be page with Project</h1>
+<div ng-controller="ProjectController" ng-init="getProject()">
+	<div class="row">
+		<!-- /PRICE ITEM -->
+		<div class="col-xs-10 col-md-7 col-xs-offset-1 col-lg-8">
+			<!--Project name and goal-->
+			<div class="item">
+				<div class="thumbnail">
+					<div ng-if="project.desc.image!=null">
+						<img src="http://placehold.it/800x450" alt="" class="centered img-responsive">
+					</div>
+					<div ng-if="project.desc.image==null">
+						<img src="http://placehold.it/800x450" alt="" class="centered img-responsive">
+					</div>
+					<div class="caption">
+						<h3>{{project.desc.name}}</h3>
+						<p>{{project.desc.goal}}</p>
+					</div>
+				</div>
+			</div>
+			<!--Description-->
+			<div class="item">
+				<div class="thumbnail">
+					<div class="caption">
+						<h3>Project description</h3>
+						<hr>
+						<p>{{project.desc.description}}</p>
+					</div>
+				</div>
+			</div>
+			<!--Category-->
+			<div class="item">
+				<div class="thumbnail">
+					<div class="caption">
+						<h3>Category</h3>
+						<hr>
+						<p>{{project.category.name}}</p>
+					</div>
+				</div>
+			</div>
+			<!--Other-->
+			<div class="item">
+				<div class="thumbnail">
+					<div class="caption">
+						<h3>Other</h3>
+						<hr>
+						<p></p>
+					</div>
+				</div>
+			</div>
+			<!--Investments-->
+			<!--todo: hide investments if user is not project owner	-->
+			<!--<div ng-if="currentUser!=null && currentUser.email == project.owner.email">-->
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)